### PR TITLE
New GQL wrapper and model fixes and starting cleanup

### PR DESCRIPTION
**Pull request by ftadros**
Note: This is populating local db only, have to move it back to prod later, prob after moving to MR and cleanup.

### CODE REVIEW

File: connector/main.go [hunk 1]
```diff
@@ -126,115 +125,74 @@ func main() {
...
+			for _, professor := range section.Instructors {
+				var rmpProfessor model.MongoProfessor
+				var finalProfessor uwomodel.Professor
+				var existingProfessor uwomodel.Professor
+				firstInitial := professor[0:1]
+				restOfName := professor[3:]
+
+				// query for the professor in the professors collection with the first initial and the rest of the name
+				// use this regexp: ^A. X$, where A=firstInitial, X=restOfName
+				regexString := "^" + firstInitial + "." + "*" + restOfName + "$"
```
Comment (mnnaegel) [inline connector/main.go:145]: think we should change the regex string to 
```suggestion
				regexString := "^" + firstInitial + "." + "*" + restOfName + "*" + "$"
```

Comment (mnnaegel) [reply connector/main.go:145]: makes the search more inclusive for peeps who have more stuff after their last name on RMP, but idk if it breaks anything or will make the search too forgiving

File: worker/constants.go [hunk 2]
```diff
@@ -11,8 +11,104 @@ const cardDepartmentSelector = "div.CardSchool__Department-sc-19lmz2k-0.haUIRO"
...
...sPageQuery(\n  $query: TeacherSearchQuery!\n  $schoolID: ID\n) {\n  search: newSearch {\n    ...TeacherSearchPagination_search_1ZLmLD\n  }\n  school: node(id: $schoolID) {\n    __typename\n    ... on School {\n      name\n    }\n    id\n  }\n}\n\nfragment TeacherSearchPagination_search_1ZLmLD on newSearch {\n  teachers(query: $query, first: 3000, after: \"\") {\n    didFallback\n    edges {\n      cursor\n      node {\n        ...TeacherCard_teacher\n        id\n        __typename\n      }\n    }\n    pageInfo {\n      hasNextPage\n      endCursor\n    }\n    resultCount\n    filters {\n      field\n      options {\n        value\n        id\n      }\n    }\n  }\n}\n\nfragment TeacherCard_teacher on Teacher {\n  id\n  legacyId\n  avgRating\n  numRatings\n  ...CardFeedback_teacher\n  ...CardSchool_teacher\n  ...CardName_teacher\n  ...TeacherBookmark_teacher\n}\n\nfragment CardFeedback_teacher on Teacher {\n  wouldTakeAgainPercent\n  avgDifficulty\n}\n\nfragment CardSchool_teacher on Teacher {\n  department\n  school {\n    name\n    id\n  }\n}\n\nfragment CardName_teacher on Teacher {\n  firstName\n  lastName\n}\n\nfragment TeacherBookmark_teacher on Teacher {\n  id\n  isSaved\n}\n"
```
Comment (mnnaegel) [inline worker/constants.go]: can we make these query strings look nicer too

File: worker/constants.go [hunk 3]
```diff
@@ -11,8 +11,104 @@ const cardDepartmentSelector = "div.CardSchool__Department-sc-19lmz2k-0.haUIRO"
...
+            name
+            id
+          }
+        }
+      }
+    }
+  }
+}
+`
+const HomePageQuery = `query TeacherSearchResultsPageQuery(
```
Comment (mnnaegel) [inline worker/constants.go]: idk what happening here.. call me pls >.>

File: worker/constants.go (hunk 2 above)
Comment (mnnaegel) [reply worker/constants.go]: we can get rid of them if we don't use it.. looks like  we dont use the get by departments anymore so

**Review by mnnaegel — APPROVED** [review]

File: worker/constants.go (hunk 2 above)
Comment (ftadros) [reply worker/constants.go]: Yep, the plan to clean all dead code next before migration.

**Pull request merged**